### PR TITLE
Bed Intersect, Search UI

### PR DESCRIPTION
**Pull request by jpfisher72**

### CODE REVIEW

File: screen2.0/src/app/search/page.tsx [hunk 1]
```diff
@@ -54,76 +59,50 @@ export default async function Search({
...
-    mainQueryParams.chromosome,
-    mainQueryParams.start,
-    mainQueryParams.end,
-    mainQueryParams.Biosample.biosample
-  )
+  //The suboptimal thing is that here would be the easy place to be able to pass the accession, but thats not possible
 
-  //Contains cell type data of the specified assembly
-  const globals: CellTypeData = await getGlobals(mainQueryParams.assembly)
+  //Main query
```
Comment (NishiPhalke) [inline screen2.0/src/app/search/page.tsx:73]: commented code can be removed

File: screen2.0/src/common/components/mainsearch/GenomicRegion.tsx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+        router.push(`/search?assembly=${assembly}&chromosome=${region.chromosome}&start=${region.start}&end=${region.end}`)
+      }
+      catch (msg) {
+        window.alert("Error in input format - " + msg)
+        setValue("")
+      }
+    }
+  }
+
+  const chromosomeLengths = {
```
Comment (NishiPhalke) [inline screen2.0/src/common/components/mainsearch/GenomicRegion.tsx:46]: can be moved to consts.ts

File: screen2.0/src/app/applets/multi-region-search/page.tsx [hunk 3]
```diff
@@ -1,12 +1,10 @@
 "use client"
 import { Typography } from "@mui/material"
-import BedUpload from "../../../common/components/BedUpload"
 
 export default function MultiRegionSearch() {
   return (
     <main>
-      <Typography variant="h6" mt={2}>Find Intersecting cCREs from BED file</Typography>
-      <BedUpload />
+      <Typography variant="h6" mt={2}>This is the Multi Region Search Page</Typography>
```
Comment (NishiPhalke) [inline screen2.0/src/app/applets/multi-region-search/page.tsx:7]: Shall we remove multi region search from applets since we have added it search ?

Comment (jpfisher72) [reply screen2.0/src/app/applets/multi-region-search/page.tsx:7]: I think I remember Jill saying that in the future we could have something similar to bed intersect but with more features. I don't remember exactly, I'll ask

Comment (NishiPhalke) [reply screen2.0/src/app/applets/multi-region-search/page.tsx]: sounds good

**Review by NishiPhalke — APPROVED** [review]

**Pull request merged**